endpointmanager/pkg/endpointlinker/endpointlinker_test.go

```diff
@@ -0,0 +1,66 @@
+package endpointlinker
+
+import (
+	"testing"
+	"strconv"
+)
+
+func Test_NormalizeOrgName(t *testing.T) {
+	orgName := "AMBULANCE & and-chair. SERVICE!"
+	expected := "AMBULANCE  AND CHAIR SERVICE"
```

Comment:
I decided against this since the strings are always tokenized on white space after this so it doesn't matter how much white space there is between tokens.